druid/src/app.rs

```diff
@@ -146,6 +148,12 @@ impl<T: Data> AppLauncher<T> {
         self
     }
 
+    /// Add additional localization resource
+    pub fn use_localization_resource(mut self, res: String) -> Self {
```

Comment:
I think this API should match the API of `L10nManager::new`; it should take a list of resources and a path explicitly, instead of us picking a hard-coded path.

I'd also include docs similar to those i n`L10nManager::new`.

Reply:
Fixed